src/main/java/radiounit_executorandreceiver/NokiaWcdmaComandExecutorFactory.java

```diff
@@ -0,0 +1,17 @@
+package radiounit_executorandreceiver;
+
+/**
+ * Responsible for the creation of NokiaWcdmaCommandExecutorFactory classes
+ * 
+ * @author esiumat
+ *
+ */
+public class NokiaWcdmaComandExecutorFactory extends RadioCommandExecutorFactory {
```

Comment:
typo: `NokiaWcdmaComandExecutorFactory` -> `NokiaWcdmaCommandExecutorFactory` (missing an 'm' in Command)

Reply:
Gotcha, thanks for catching that